var/spack/repos/builtin/packages/py-wurlitzer/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyWurlitzer(PythonPackage):
+    """Capture C-level stdout/stderr pipes in Python via os.dup2."""
+
+    homepage = "https://github.com/minrk/wurlitzer"
+    url      = "https://files.pythonhosted.org/packages/e8/2c/3e57755689fcf75aa25f6afba064d3891e9864ef43a24745575c86b12ad4/wurlitzer-3.0.2.tar.gz"
```

Comment:
This hash won't work as soon as we add another version: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi

Reply:
Arg sorry clearly I need to read the instructions more carefully

Reply:
Ah I see that it was in an entirely different section of the (very long) package instructions. I'll add a PR to cross-reference these and update the default python package template.

Reply:
If you `spack create https://super.long.pypi/url` it will automatically create a `pypi = 'name/name-version.ext'` line for you, so most users don't run into this.

Reply:
Aha, that makes sense. I've only installed this one through Conda and the github page seemed more official since it was referenced by the pypi site. Newbie mistake :)